package.json

```
@@ -11,7 +11,7 @@
 		"url": "https://sindresorhus.com"
 	},
 	"engines": {
-		"node": ">=8"
+		"node": "^10.17 || >=11.14"
```

```suggestion
		"node": "^10.17"
```

11 is not a supported version anyway.